source/ui/x11/UnixWindow.ooc

```diff
@@ -74,11 +74,11 @@ UnixWindow: class extends UnixWindowBase {
 				map add("texture0", matchedImage)
 		}
 		sourceSize := image size toFloatVector2D()
-		map textureTransform = GpuCanvas _createTextureTransform(image size, IntBox2D new(image size))
-		map model = FloatTransform3D createScaling(sourceSize x / 2.0f, sourceSize y / 2.0f, 0.0f)
-		map view = this _openGLWindow _createView(sourceSize, FloatTransform3D identity)
-		map projection = this _openGLWindow _createProjection(sourceSize, 0.0f)
-		map use(null)
+		textureTransform := GpuCanvas _createTextureTransform(image size, IntBox2D new(image size))
+		model := FloatTransform3D createScaling(sourceSize x / 2.0f, sourceSize y / 2.0f, 0.0f)
+		view := this _openGLWindow _createView(sourceSize, FloatTransform3D identity)
+		projection := this _openGLWindow _createProjection(sourceSize, 0.0f)
```

Comment:
Transforms are stored in temporary variables and then multiplied. Anything that is not used can be removed from the transform product.